modules/tlc2/overrides/IOUtils.java

```diff
@@ -74,14 +84,131 @@ public static final IValue serialize(final IValue value, final StringValue absol
 		return BoolValue.ValTrue;
 	}
 	
+	/* Writes a String as plain text to file.
+	 * Operator should be called as Serialize(payload, filepath, [ser |-> "TXT", openOption |-> openOption, charset |-> charset])
+	 *		String payload: is the string that will be written
+	 *      String filepath: is the file where the string will be written
+	 *      String openOptions: sequence of strings of java StandardOpenOptions
+	 *      String charset: string with a java standard charset
+	 *      
+	 * Example:
+	 * 		Serialize("test payload", "test.txt", [ser |-> "TXT", charset |-> "UTF-8", openOption |-> <<"WRITEA", "CREATE", "TRUNCATE_EXISTING">>])
+	 */
+	@Evaluation(definition = "Serialize", module = "IOUtils", warn = false, silent = true, priority = 50)
+	public synchronized static Value textSerialize(final Tool tool, final ExprOrOpArgNode[] args, final Context c,
+			final TLCState s0, final TLCState s1, final int control, final CostModel cm) {
+		
+		final String msgInvalidParam = "Serialize error invalid parameters: ";
+		final String successmsg = "Finish writting to the file with success!";
+		
+		final RecordValue opts;	
+		
+		try {
+			opts = (RecordValue) tool.eval(args[2]);
```

Comment:
This should pass the context, s0, s1... to eval: https://github.com/tlaplus/CommunityModules/pull/52/commits/0f53203e8a52d46097979140e1885c30684f1ec4#diff-68c25ed4bd43494aa2d8b637bde5337432cf264c28185423dd9e8418a8ac98e8R264

Reply:
Add it in 76d03d5